### PR TITLE
feat: cc-0031 modify ruler btn

### PR DESCRIPTION
**Pull request by stevenliu986**

### CODE REVIEW

File: src/__tests__/components/EditorFooter/index.test.tsx [hunk 1]
```diff
@@ -7,14 +7,15 @@ describe("EditorFooter", () => {
...
-  });
+  // caz Ruler off/on no need
+  // it("should toggle between on and off", () => {
+  //   const { getByTestId } = render(<EditorFooter />);
+  //   const switchButton = getByTestId("switch-btn");
+  //   fireEvent.click(switchButton);
+  //   expect(getByTestId("ruler-label")).toHaveTextContent("Ruler off");
+  //   fireEvent.click(switchButton);
+  //   expect(getByTestId("ruler-label")).toHaveTextContent("Ruler on");
+  // });
```
Comment (Fuchih) [inline src/__tests__/components/EditorFooter/index.test.tsx:18]: seems these codes don't need anymore you can delete it without hesitate

**Review by Fuchih — APPROVED** [review]

File: src/components/EditorFooter/index.tsx [hunk 2]
```diff
@@ -38,25 +36,22 @@ const EditorFooter = () => {
...
           >
-            {rulerLabel}
+            Ruler
           </FormLabel>
           <Switch
-            id="ruler-swith-btn"
+            id="ruler-switch-btn"
             colorScheme="footerSwitch"
             sx={{
               "span:first-child": {
```
Comment (kritomomo) [inline src/components/EditorFooter/index.tsx:50]: image

**Review by tatoMa — COMMENTED** [review]
Give it a try using the global CSS modifier but aimed at this specific switch thumb color changing.
image
image

**Review by tatoMa — APPROVED** [review]
LGTM

File: src/components/EditorFooter/index.tsx [hunk 3]
```diff
@@ -38,26 +38,28 @@ const EditorFooter = () => {
...
           </FormLabel>
           <Switch
-            id="ruler-swith-btn"
+            id="ruler-switch-btn"
             colorScheme="footerSwitch"
+            className="custom-switch"
             sx={{
               "span:first-child": {
                 bgColor: "brand.primary",
               },
```
Comment (tatoMa) [inline src/components/EditorFooter/index.tsx:56]: No first-child selector anymore. Please.

**Review by tatoMa — CHANGES_REQUESTED** [review]

File: src/components/EditorFooter/index.tsx (hunk 2 above)
Comment (kritomomo) [reply src/components/EditorFooter/index.tsx:54]: https://github.com/emotion-js/emotion/issues/1178

File: src/components/EditorFooter/index.tsx (hunk 3 above)
Comment (stevenliu986) [reply src/components/EditorFooter/index.tsx]: changed the selector.

**Review by tatoMa — APPROVED** [review]
LGTM. If nobody uses the global CSS style, you can simply delete it.

**Review by tatoMa — APPROVED** [review]

**Review by tatoMa — APPROVED** [review]
LGTM

**Pull request merged**